keyboards/kindakeyboards/conone65/keymaps/via/readme.md

```diff
@@ -0,0 +1,25 @@
+# Conone65
+
```

Comment:
I didn't add an Imgur link to the keyboard which was in the template, but I have asked the keyboard designer if I can reuse one of the images from the Kindakeyboards website. Is the image absolutely required before merging?

Reply:
Not required, but helpful to have, specifically of the PCB. Be sure to host it on Imgur if you are able to add an image.

Reply:
Thanks, I'll add an image and update in the next few days.

Reply:
I've added an Imgur link for the PCB.